config/build/webpack.conf.js

```diff
@@ -93,6 +94,9 @@ module.exports = (mode) => {
 	 * List of extensions that webpack should try to resolve
 	 */
 	wpConfig.resolve = {
+		alias: {
```

Comment:
Can you throw a one liner comment above this property that explains what this is achieving?